test_conformance/compiler/test_build_options.cpp

```diff
@@ -93,31 +90,42 @@ REGISTER_TEST(options_build_optimizations)
     int error;
     cl_build_status status;
 
-    for(size_t i = 0; i < sizeof(optimization_options) / (sizeof(char*)); i++) {
+    Version version = get_device_cl_version(device);
 
+    for (size_t i = 0; i < sizeof(optimization_options) / (sizeof(char *)); i++)
```

Comment:
for loop @ line 95 computes size incorrectly now that optimization_options is an array of std::pair. Makes test segfault:

Reply:
Done.
As we are using C++17, I use a more compact declaration of the array.

Reply:
I was just missing the `#include <array>` 😅